project/plugins.sbt

```diff
@@ -14,4 +14,6 @@ libraryDependencies += "org.vafer" % "jdeb" % "1.3" artifacts Artifact("jdeb", "
 // These are for releasing to Sonatype
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.2")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.10")
\ No newline at end of file
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.10")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
I used this during testing, figure I may as well leave it here. I don't feel strongly about this though, happy to remove it?

Reply:
I would lean towards this going in your local ~/.sbt/ plugins settings rather than at a project level, not a big deal though

Reply:
Thanks @philmcmahon, you're absolutely right. That would be a much better approach, I'll remove this today.